### PR TITLE
Add cmake support

### PR DESCRIPTION
**Pull request by jseward**
PR for fixing https://github.com/PlayFab/gsdk/issues/69

Also fix bad path to json header.

### CODE REVIEW

**Comment by jseward** [comment]
Needed for https://github.com/microsoft/vcpkg/issues/17477

**Review by AustinBrunkhorst — APPROVED** [review]

File: cpp/cppsdk/gsdkLinuxPch.h [hunk 1]
```diff
@@ -3,5 +3,5 @@
 
 #pragma once
 
-#include "jsoncpp/json/json.h"
+#include "json/json.h"
```
Comment (dgkanatsios) [inline cpp/cppsdk/gsdkLinuxPch.h:6]: thanks for the PR! Does this work in Windows as well?

Comment (jseward) [reply cpp/cppsdk/gsdkLinuxPch.h:6]: Yes it's a linux only precompiled header include. You can see that `gsdkWindowsPch.h` has the same path. But perhaps I'm missing something? `jsoncpp/` folder does not exist anywhere.

**Comment by dgkanatsios** [comment]
merged, thank you!

**Pull request merged**